### ISSUE TITLE
Normalize brand names

### ISSUE THREAD

**Issue by Romeo-PHILLIPS** [alltheplaces/alltheplaces#4104]
Hi,

just writing this idea down (maybe I will try to implement it in the future or if someone else thinks its valuable they can). Currently we don't have any "normalization" on brand names and I think it should be done. 

One of the first step in my opinion should be if a spider has a wiki-data linked to it in the `item_attributes` dictionary, we should pull the brand name from wiki-data and set it inside the item-attributes (even override it) if its set.

One example could be the domino's scraper use the brand name "Domino's" but the wiki-data calls it "Domino's Pizza"

**Comment by Cj-Malone** [comment]
> One of the first step in my opinion should be if a spider has a wiki-data linked to it in the item_attributes dictionary, we should pull the brand name from wiki-data and set it inside the item-attributes

I agree in principle, I don't know if we set NSI or Wikidata as the canonical brand. Actually we can now apply `brand` from NSI when we get a match in `ApplyNSICategoriesPipeline`, assuming the item has `brand_wikidata`, (and no `brand` as we don't override).

> (even override it if its set)

I don't think I agree, but I'm open to be convinced.

> One example could be the domino's scraper use the brand name "Domino's" but the wiki-data calls it "Domino's Pizza"

Another one that is about to be highlighted in https://github.com/alltheplaces/alltheplaces.xyz/pull/23, is `McDonald's` vs `McDonald’s`, Wikidata has the fancy apostrophe, we don't.

**Comment by Romeo-PHILLIPS** [comment]
> I don't think I agree, but I'm open to be convinced.

My point is that we should follow a census. I don't consider the variable set by a developer in a spider to be considered to follow a global census, making the brand name only "normalized" inside the developers spider and harder to match it inside other projects. 

In my opinion, if the spider has a wikidata linked to it, we should use the brand it gives us and nothing else.

**Comment by iandees** [comment]
I'm not ok with overwriting an ATP developer's settings from outside the project, but I am OK with it getting flagged and fixed in ATP. My preferred fix is probably to remove the value in ATP so that it's set from the preferred "source of consensus" (in this case: NSI).
